frameworks/kafka/cli/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Would it make sense to link this to the `sdk/cli/build.sh` version of the file? May not be worth the effort though.